staff/francis-hrom/strings/strings.js

```diff
@@ -0,0 +1,156 @@
+// https://trello.com/c/vKRdfoi4/32-feature-strings
+
+// https://pluscoders.com/javascript/strings/challenges/spell/
+function spell(text) {
+  for (letter of text) {
+    console.log(letter);
+  }
+}
+
+// https://pluscoders.com/javascript/strings/challenges/string-to-array/
+function stringToArray(text) {
+  let array = [];
+  for (letter of text) {
+    array.push(letter);
+  }  
+  return array;
+} 
+
+// provide other ways to resolve the string-to-array in other versions of the script (string-to-array.x.js) may you investigate a bit ,)
+// Option 1: string.split('');
+function stringToArray(text) {
+  return text.split('');
+} 
+
+// Option 2: [...string];
+function stringToArray(text) {
+  return [...text];
+} 
+
+// Option 3: Array.from(string);
+function stringToArray(text) {
+  return Array.from(text);
+} 
+
+// Option 4: Object.assign([], string);
+function stringToArray(text) {
+  return Object.assign([], text);
+} 
+
+// https://pluscoders.com/javascript/strings/challenges/concat/
+function concat(...texts) {
+  let str = "";
+  for (text of texts) {
+    str += text;
+  }
+  return str;
+}
+
+
+function concat(...strings) {
+  return strings.join('');
+}
+
+
+// https://pluscoders.com/javascript/strings/challenges/repeat/
+function repeat(text, count) {
+    str = "";
+    for (let i=0; i< count; i++) {
+      str += text;
+    };
+    return str;    
+}
+
+// alternative solution
+function repeat(text, count) {
+    return text.repeat(count);
+}
+
+// https://pluscoders.com/javascript/strings/challenges/search/
+function search(text, pattern) {
+  return text.search(pattern);
+}
+
+// another solution for string only, regex would not work in this scenario
+function search(text, pattern) {
+  return text.indexOf(pattern);
+}
+
+// another solution for string only, regex would not work in this scenario
+function search(text, pattern) {
+  let pos = -1;
+  for (let i=0; i < text.length-pattern.length; i++) {
+    if (text.substr(i,pattern.length) == pattern) {
+      pos = i;
+    }
+  }
+  return console.log(pos);
```

Comment:
what happens in this line, are you really returning the matching string? pay attention to it ,)